code/modules/reagents/reagent_dispenser.dm

```diff
@@ -513,6 +525,17 @@
 	desc = "Originally used to store liquids & powder. It is now used as a source of comfort. This one is made of wood."
 	health = 30
 
+/obj/structure/reagent_dispensers/cauldron/barrel/wood/attackby(obj/item/weapon/W, mob/user)
+	if (iscrowbar(W))
+		var/obj/item/tool/crowbar/C = W
+		to_chat(user, "<span class='notice'>Now disassembling table...</span>")
+		C.playtoolsound(src, 50)
+		if(do_after(user, src,50))
+			var/obj/item/stack/sheet/wood/WS = new(src.loc, 20)
+			WS.forceMove(src.loc)
+			qdel(src)
+		return
```

Comment:
no trailing return
forgot to add ..() might now be impossible to get reagents out of this dispenser
the forcemove isn't necessary because it's already being created at src.loc
you don't need src. at all, just use loc
your notice message refers to the barrel as a table